manifests/node-maintenance-operator/0.1.0/node-maintenance-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,163 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Not sure why there are two clusterserviceversions. Are these two equivalent?

Reply:
not for 100%. One clusterversion file is created by operator-sdk and is saved in deploy/olm-catalog/node-maintenance-operator/<version>/ and then it is updated by update-olm.py script and saved into manifests/node-maintenance-operator/<version>/

Reply:
The logic is the following:
- manifests under `deploy/` are the "pristine" or the master sources of truth. This involves hand-crafted files, or files created by automated tools. Everything else is created from these.
- manifests under `manifests/` are to be used for distribution (e.g. releases, bundles)

operator-sdk suggests/requires manual edits, done by `update-olm.py` script. I choose to
generate the "master" CSV under `deploy`, and create the edited, final version under `manifests`.

We are reviewing the manifests layout in another patch to make it less confusing.